infra/ansible/stage_inventory.yaml

```diff
@@ -1,10 +1,9 @@
 aggregator:
   hosts:
     aligned-holesky-aggregator-1:
-      ansible_host: aligned-ansible
-      ansible_user: admin
-      app_user: app
-      ansible_python_interpreter: /usr/bin/python3
+      ansible_host: aligned-holesky-stage-1-aggregator
+      admin_user: admin
+      ansible_user: app
```

Comment:
There is missing ansible_python_interpreter: /usr/bin/python3

Reply:
Ditto. https://github.com/yetanotherco/aligned_layer/commit/a1d5e4e8e3b71accc0b2bc3dabbd5dfbd8c72805